addons/field-guide/addon/controllers/show.js

```diff
@@ -2,22 +2,90 @@ import Controller from '@ember/controller';
 import showdown from 'showdown';
 import config from 'ember-get-config';
 
+const mapElementsToClassNames = {
+  // TODO decide if shorten them to `doc-md-[...]`
```

Comment:
I'm ok with either but I do think spelling out the full word makes it easier to understand the class names' purpose at a glance.

Reply:
Idea: Maybe make a variable for the prefix ("doc-markdown-") in case you want to change it later?

Reply:
OK, I'll use `doc-markdown` for now (also Alex prefers this).